stdlib/builtin/regexp.rbs

```diff
@@ -38,13 +39,16 @@ class Regexp < Object
   # Produce a nicely formatted string-version of *rxp* . Perhaps
   # surprisingly, `#inspect` actually produces the more natural version of
   # the string than `#to_s` .
-  # 
+  #
   # ```ruby
   # /ab+c/ix.inspect        #=> "/ab+c/ix"
   # ```
   def inspect: () -> String
 
-  def match: (String? arg0, ?Integer arg1) -> MatchData?
+  def match: (String arg0, ?Integer arg1) -> MatchData?
```

Comment:
`Regexp#match` and `match?` receive a symbol, nil, and object that has `to_str`.
So I think `arg0` should be a union type of them.

Reply:
FYI https://github.com/ruby/ruby-signature/blob/ba26445a4f18c26d140f201896d7d1ace7b3c717/stdlib/builtin/builtin.rbs#L13-L15

Reply:
Thanks for your comment! 😄 
I fixed in https://github.com/ruby/ruby-signature/pull/75/commits/8f9ce2a3105313338eb4aacb7d02118e63aacde4 👍